### PR TITLE
Switch to using OpenAstronomy workflow for publishing wheels

### PR DESCRIPTION
**Pull request by astrofrog**
This should simplify the configuration - for now I'm explicitly listing different targets in case some don't work, will simplify further once I see what works/doesn't work.

(needs rebasing once #50 is merged)

### CODE REVIEW

**Comment by astrofrog** [comment]
@keflavich - this should be good to go assuming the 32-bit build passes. Once this is merged, I could try and tag a release?

**Comment by keflavich** [comment]
Yes, that would be great!

**Review by keflavich — APPROVED** [review]

**Pull request merged**
